controllers/default.py

```diff
@@ -615,7 +616,7 @@ def leaderboard():
 
     if request.vars.has_key("q") and request.vars["q"]:
         heading = T("Institute Leaderboard")
-        institute = request.vars["q"]
+        institute = urllib.unquote(request.vars["q"])
```

Comment:
please import urllib just before this statement instead of across the controller

Reply:
@raj454raj changed the import location of the library